src/shared/sol-util-file.c

```diff
@@ -26,6 +26,7 @@
 #include <fcntl.h>
 #include <libgen.h>
 #include <limits.h>
+#include <sched.h>
```

Comment:
On 06-04-2016 11:23, Gustavo Lima Chaves wrote:
> In src/shared/sol-util-file.c
> <https://github.com/solettaproject/soletta/pull/1715#discussion_r58713578>:
>
>> @@ -26,6 +26,7 @@
>>  #include <fcntl.h>
>>  #include <libgen.h>
>>  #include <limits.h>
>> +#include <sched.h>
>
> ifdef under linux as well?

Ouch... true. Tks!

>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta/pull/1715/files/0d20a392a88ad51443271fd254faf37a46739bd0#r58713578>
>